packages/atomic/cypress/e2e/facets/color-facet/color-facet.cypress.ts

```diff
@@ -378,6 +378,55 @@ describe('Color Facet Test Suites', () => {
     });
   });
 
+  describe('with #resultsMustMatch set to default value', () => {
```

Comment:
Notice that most tests only check the request. Would there be value in adding any checks on the UI state (and results etc) or are they taken care of by other existing tests?

Reply:
It's kinda hard to test the UI based on this option since there is no logic in atomic regarding the filtering logic. The `resultsMustMatch` option is tested in headless.